R/covr.R

```diff
@@ -44,7 +58,10 @@ trace_calls <- function (x, srcref = NULL) {
       fun_body <- trace_calls(fun_body)
     }
 
-    formals(x) <- trace_calls(formals(x))
+    #formals(x) <- trace_calls(formals(x))
```

Comment:
I do not remember exactly, but it is caught by one of my new tests, whith a no-argument function. In fact formals(f) <- NULL crashes when the formals were already NULL.